update-vo-config/setup.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
@Pansanel thanks for maintaining this tool. Personnally I don't consider as a good practice to put `python3` rather tha `python` as `python3` is not always available,. In particular on Windows where there is no symlinks. For the the good practice is to use an `assert`. Something like:

```python
try:
    assert sys.version_info >= (3, 8)
except AssertionError:
    print("Error: Python 3.8+ required.")
    sys.exit(1)
```
(adjust the requirement to whatever you consider the appropriate value)

Reply:
Thanks, @jouvin for reviewing this PR. I have updated the scripts as suggested. The tool need to load urllib.request, that is only available on Python 3. In consequence, I have linked the ImportError with the use of Python 2.